canonical-web-and-design/jaas-dashboard
# Grouping by cloud

## Done

Add grouping by cloud

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Click the "Cloud" filter and verify that the models switch to being filtered by "Cloud" 

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1072
